lua/tardis/settings/sh_miscellaneous.lua

```diff
@@ -247,6 +247,19 @@ TARDIS:AddSetting({
     name="NoInterior",
 })
 
+TARDIS:AddSetting({
+    id="legacy_door_type",
```

Comment:
This can be a user personal preference and doesn't need to be a spawn and "TARDIS-wide" setting for everyone - is there any reason it was done this way in particular?

Reply:
It would be a mess when one can't open the doors and someone else can

Also, one of the use-cases I see is when portals are too laggy for someone (and they don't want that black replacement)

I think it's for the best for the TARDISes to be of certain "types"
It either has portals or a legacy-style entrance

Reply:
Everyone can spawn their own kind, and if you changed your mind you can just respawn it with a different type

Reply:
Discussed on Discord, happy with the approach